tests/data_context/store/test_store_dynamic_loading.py

```diff
@@ -5,7 +5,7 @@
 from great_expectations.util import verify_dynamic_loading_support
 
 
-@pytest.mark.integration
+@pytest.mark.unit
```

Comment:
This one is a real change. Should have been marked as `unit`, since we just check that the module names end with the correct string